app/src/main/java/de/westnordost/streetcomplete/data/osmnotes/notequests/OsmNoteQuestController.kt

```diff
@@ -91,7 +92,7 @@ import javax.inject.Singleton
 
     private fun createQuestForNote(note: Note, blockedNoteIds: Set<Long> = setOf()): OsmNoteQuest? =
         if(note.shouldShowAsQuest(userStore.userId, showOnlyNotesPhrasedAsQuestions, blockedNoteIds))
-            OsmNoteQuest(note.id, note.position)
+            OsmNoteQuest(note.id, note.position.toOsmLatLon())
```

Comment:
The data classes from `de.westnordost.osmapi` should not occur at all in the StreetComplete source code anymore except in the `NotesApiImpl` class